content/research/papers.md

```diff
@@ -3,153 +3,107 @@ title: "Papers"
 date: 2021-05-28T09:27:27-05:00
 draft: false
 layout: page
+aliases: /research/
 ---
-## Papers using the *tardis* package
 
-**Dutta, Anirban, Singh, Avinash, Anupama, G. C., et al.** 2021, MNRAS, *"SN 2017hpa: a carbon-rich Type Ia supernova"* <a href="https://ui.adsabs.harvard.edu/abs/2021MNRAS.503..896D" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+## Papers using the _tardis_ package
 
```

Comment:
I'm afraid I don't quite understand why this change(and multiple following) was done. Did you run some sort of formatter?

Reply:
My VS Code is set to format on save. Maybe that is why it happened.